--- base/base_config	2024-08-30 08:38:25.341989847 +0300
+++ config	2024-08-30 10:00:19.052885466 +0300
@@ -3,12 +3,21 @@
     "position": "bottom",
     "modules-left": ["sway/workspaces", "sway/mode"],
     "modules-center": ["sway/window"],
-    "modules-right": ["clock"],
+    "modules-right": ["battery", "clock"],
     "sway/window": {
         "max-length": 50
     },
+    "battery": {
+        "format": "{capacity}% {icon}",
+        "format-icons": ["", "", "", "", ""]
+    },
     "clock": {
         "format-alt": "{:%a, %d. %b  %H:%M}"
+    },
+    "backlight": {
+        "device": "intel_backlight",
+        "format": "{percent}% {icon}",
+        "format-icons": ["", ""]
     }
 }
 
